.mypy.ini

```diff
@@ -0,0 +1,24 @@
+[mypy]
+files = async_lru, tests
+check_untyped_defs = True
+follow_imports_for_stubs = True
+disallow_any_decorated = True
```

Comment:
Do you have a reason for enumerating all mypy options?
I would prefer `strict = true` plus maybe a few additional fine-tunes.
Maintaining all mypy opts adds a burden; I never could keep all these options in my mind without looking in docs

Reply:
This adds a lot more options than strict. This is the config we started in aiohttp a few years ago. Basically, strict isn't so strict, so we like to enable all the options that aren't too much.

At some point I'd like to find a way to make a config that's reusable across all projects without copy/paste, which will ease the maintenance burden.